test/system/src/main/java/io/pravega/test/system/framework/Utils.java

```diff
@@ -231,4 +231,9 @@ private static boolean isTLSEnabled() {
         String tlsEnabled = Utils.getConfig("tlsEnabled", "false");
         return Boolean.valueOf(tlsEnabled);
     }
+
+    public static boolean isSkipLogDownloadEnabled() {
+        String config = getConfig("skipLogDownload", "false");
+        return config.trim().equalsIgnoreCase("true") ? true : false;
```

Comment:
Wouldn't just `return config.trim().equalsIgnoreCase("true")` be equivalent?

Reply:
Yes. I just followed the above calls for consistency purposes.